docs/csharp/language-reference/compiler-messages/cs0116.md

```diff
@@ -11,7 +11,7 @@ ms.assetid: 4cb137b5-ec29-4c1a-adde-9f8424cb9496
 
 A namespace does not directly contain members such as fields or methods  
   
- Inside a [namespace](../../../csharp/language-reference/keywords/namespace.md), the compiler only accepts classes, structures, unions, enumerations, interfaces, and delegates. This error is often generated by developers from a C/C++ background who forget that in C#, methods and variables must be declared and defined within a struct or class. For more information, see [General Structure of a C# Program](../../../csharp/programming-guide/inside-a-program/general-structure-of-a-csharp-program.md).  
+ Inside a [namespace](../keywords/namespace.md), the compiler only accepts namespaces, classes, structures, enumerations, interfaces, and delegates. This error is often generated by developers from a C/C++ background who forget that in C#, methods and variables must be declared and defined within a struct or class. For more information, see [General Structure of a C# Program](../../../csharp/programming-guide/inside-a-program/general-structure-of-a-csharp-program.md).  
```

Comment:
Perhaps it would be better to simply say that it only accepts namespaces and type definitions rather than enumerating the various kind of types C# has. If new types are added to C# (e.g. records) then this kind of description will get stale.

Reply:
@mikedn that's a good idea; thanks for sharing!